packages/core/dist/components/Review/Review.js

```diff
@@ -115,7 +117,11 @@ Review.propTypes = {
    * An optional function that is executed on edit link click. The event and
    * props.editHref value are passed to this function.
    */
-  onEditClick: _propTypes2.default.func
+  onEditClick: _propTypes2.default.func,
+  /**
+   * An optional node in place of the edit link. If this defined, no edit link will be shown.
+   */
+  editContent: _propTypes2.default.node
```

Comment:
You don't need to include the dist files in your PR because they will be built when a release is created.

Reply:
gotcha!